docs/docs/docs/developer-resources/testing.md

```
@@ -59,6 +59,64 @@
+```ts
+import { createMockGraphQLContext } from "test/MockContext/mockGraphQLContext";
+```
+
+### **Creating a Mock Context**  
+#### **For an Unauthenticated User**  
+```ts
+const mockContext = createMockGraphQLContext(false);
+```
+`mockContext.currentClient.isAuthenticated` will be `false`.  
+
+#### **For an Authenticated User**  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)